.github/workflows/test.yml

```diff
@@ -12,30 +12,25 @@ on:
     branches:
       - main
     paths-ignore:
-      - 'packages/react-native-sdk/docusaurus/**'
-      - 'packages/react-sdk/docusaurus/**'
+      - '!**/docusaurus/**'
```

Comment:
```suggestion
      - '**/docusaurus/**'
```

Reply:
This is what happens when copy-pasting 😅 Thanks for catching this!